app/assets/javascripts/google-maps.js

```diff
@@ -0,0 +1,91 @@
+
+// Create a map with a marker and center the map at the marker.
```

Comment:
I wonder if it would be best to load this from a CDN.  Depending on where we host the prod server (not sure where that is but doubt it is in Sweden) a CDM will almost always deliver files faster than we can from our hosting environment.